src/routes/index.svelte

```diff
@@ -138,12 +138,24 @@
         }
     }
 
-    function shareResults() {
+    function shareResults(e) {
```

Comment:
This parameter should be named `h` rather than `e`, since `h` is an objectively superior letter.